components/istio-kyma-patch/README.md

```diff
@@ -2,38 +2,35 @@
 
 ## Overview
 
-To run kyma some changes needs to be made in default Istio isntallation. This application patches already existing Istio 
-installation of Istio so it can be used by kyma.
+To run Kyma, the default Istio installation needs some changes. This application patches the already existing Istio 
+installation so that Kyma can use Istio.
 
-The patch application performs several steps:
-1. Check if required CRDs are deployed on setup. The script reads file named `required-crds`. The file should contain list of
-Istio's CRDs which are required by kyma. 
+## Usage
+
+The application performs several steps:
+1. Check if the required CRDs are already deployed. The script reads the `required-crds` file which should contain a 
+list of Istio's CRDs that Kyma requires. If CRDs are not deployed patch will fail.
 
-1. The configuration of sidecar injector. The following changes are introduced to the `istio-sidecar-injector` ConfigMap:
+2. Configure the sidecar injector. The following changes are introduced to the `istio-sidecar-injector` ConfigMap:
     * The **policy** parameter is set to `disabled`.
-    * The **zipkinAddress** points to zipkin deployed in `kyma-system` Namespace.
-    * All containers have set default **resources.limits.memory** to `128Mi` and **resources.limits.cpu** to `100m`.
+    * The **zipkinAddress** points to Zipkin deployed in the `kyma-system` Namespace.
+    * All containers have the default **resources.limits.memory** set to `128Mi` and **resources.limits.cpu** to `100m`.
 
-1. Istio components are patched. The script is expecting files in format `<resource-name>.<kind>.patch.json` which contain
-patch in JsonPatch format. Components are patched using `kubectl patch` command. Patched resource may not exist. 
-If other failure occurs the script must fail. See [job ConfigMap](../../resources/istio-kyma-patch/templates/configmap.yaml) 
-to see which patches are applied by default.
+3. Patch Istio components. The script looks for files in the `{resource-name}.{kind}.patch.json` format which contain a 
+`JsonPatch`. The components are applied using the `kubectl patch` command. The modified resource may not exist. Patch 
```

Comment:
`The modified resource may not exist, in which case the patch is skipped. See the [job ConfigMap](../../resources/istio-kyma-patch/templates/configmap.yaml) to learn which patches are 
applied by default. On any other failure, such as wrong patch format or network error, the application fails.`